assemblyai-starter/README.md

```diff
@@ -0,0 +1,47 @@
+# AssemblyAI Starter
+
+This is an Encore starter for a Conversational Intelligence app that uses [AssemblyAI](https://assemblyai.com). It lets you upload and transcribe voice data.
+
+## Developing locally
```

Comment:
Please add a section that briefly describes the secrets that the user needs to set (`AssemblyAIAPIKey` & `AssemblyAIWebhookSecret`). Please give a short explanation why and when the webhook is needed.

Here is an example: https://github.com/encoredev/examples/tree/main/ts/elevenlabs#elevenlabs-api-key

Reply:
Also, I get `unable to submit file for transcription: Authentication error, API token missing/invalid` when I try to upload a file. I have created an AssemblyAI account and set the API key as the secret. Any idea why that happens? Probably good to add to this section if so.

Reply:
Please mention that the app does not work locally because it relies on webhooks. 

You could add an endpoint to to check the status of a transcript and then add a button in the UI so that it's possible to check if the status has changed 🤷 Then it would work locally as well, which would be nice.